Tasks/AzureRmWebAppDeployment/kuduutility.ts

```diff
@@ -297,25 +332,27 @@ async function getFileContent(publishingProfile, physicalPath, fileName) {
     };
     var kuduGetFileUrl = "https://" + publishingProfile.publishUrl + "/api/vfs/" + physicalPath + "/" + fileName;
     tl.debug('Getting content of file: ' + fileName + ' using publishUrl: ' + kuduGetFileUrl);
-    httpObj.get('GET', kuduGetFileUrl, headers, (error, response, body) => {
-        if(error) {
-            defer.reject({ error: error + ''});
-        }
-        else if(response.statusCode === 200) {
+    let options: rm.IRequestOptions = {};
+    options.additionalHeaders = headers;
+    let promise: Promise<any> = hc.get(kuduGetFileUrl, options);
+    promise.then(async (response) => {
+        let contents: string = await response.readBody();
```

Comment:
This call should be under a try-catch. If not then any exception thrown wont be notified.